src/api/v1beta1/properties_test.go

```diff
@@ -441,7 +441,7 @@ func TestDynaKube_ShallUpdateActiveGateConnectionInfo(t *testing.T) {
 		},
 	}
 
-	timeProvider := timeprovider.New()
+	timeProvider := timeprovider.New().SetToNow()
```

Comment:
I'm somehow not entirely happy with the naming. OTOH I do not have an superior idea. Maybe
```suggestion
	timeProvider := timeprovider.New().Freeze()
	timeProvider := timeprovider.New().Lock()
```
IDK...

Reply:
`SetToNow` renamed to `Freeze`